debian/homestead.postinst

```diff
@@ -89,13 +89,6 @@ case "$1" in
 
         service clearwater-infrastructure restart
 
-        # Stop the cluster manager, so that it is restarted by Monit and picks
-        # up the new Cassandra scaling plugin. We check whether the process is 
-        # installed to avoid warning logs when etcd isn't being used
-        if [ -x "/etc/init.d/clearwater-cluster-manager" ]; then
-          service clearwater-cluster-manager stop || /bin/true
-        fi  
-
```

Comment:
I did discuss with Rob, and we decided to remove this for a reason that wasn't to do with split-storage.  If you look here https://github.com/Metaswitch/clearwater-cassandra/blob/master/debian/clearwater-cassandra.postinst#L72 you'll see that we've already stopped the cluster manager in clearwater-cassandra so we don't need to do it again here.  I suspect that this code is from before cassandra was in its own repo and we forgot to remove it.